README.md

```diff
@@ -88,9 +88,10 @@ var options = {
 };
 ```
 
-#### `uncaughtException` (*Function*, optional)
+#### `uncaughtException` (*Function*, optional) **DEPRECATED**
 Handler for `uncaughtException` errors. See the [endgame](https://github.com/totherik/endgame) module for defaults.
 
+**Deprecated**: See `shutdown` middleware instead.
```

Comment:
I don't want to hold this up any longer so I'm going to merge but I think we need to think about how we want to handle this before we cut a release.